github/PaginatedList.py

```diff
@@ -88,7 +90,7 @@ def __finished(self, index):
             return self.__stop is not None and index >= self.__stop
 
 
-class PaginatedList(PaginatedListBase):
+class PaginatedList(PaginatedListBase, Generic[T]):
```

Comment:
this generic is not actually used in this pr？

Reply:
resolved